.teamcity/_self/projects/WPComTests.kt

```diff
@@ -214,12 +219,14 @@ fun gutenbergBuildType(screenSize: String, buildUuid: String): BuildType {
 	}
 }
 
-fun gutenbergPlaywrightBuildType( targetDevice: String, buildUuid: String ): E2EBuildType {
+fun gutenbergPlaywrightBuildType( targetDevice: String, buildUuid: String, atomic: Boolean = false ): E2EBuildType {
+	var isAtomicDescription = if (atomic) "atomic" else "";
+
     return E2EBuildType (
-		buildId = "WPComTests_gutenberg_Playwright_$targetDevice",
+		buildId = "WPComTests_gutenberg_Playwright_${isAtomicDescription}_$targetDevice",
```

Comment:
This actually won't work well. It'll generate WPComTests_gutenberg_Playwright__desktop" if not-atomic. I'll fix it.